appraisal.lic

```diff
@@ -10,25 +10,35 @@ class Appraisal
 
   def initialize
     @equipment_manager = EquipmentManager.new
-    @equipment_manager.wear_equipment_set?('standard')
     @settings = get_settings
+    train_list = get_settings.appraisal_training
```

Comment:
This should be `train_list = @settings.appraisal_training`. Although, the only place `@settings` is even used is `train_appraisal_with_pouches`. You could make `full_pouch_container` an argument to that method and then `@settings` doesn't have to be an instance var.